qadence/ml_tools/constructors.py

```diff
@@ -724,6 +731,12 @@ def build_qnn_from_configs(
     fm_config: FeatureMapConfig,
```

Comment:
Would it make sense to:

```suggestion
    fm_config: FeatureMapConfig = FeatureMapConfig(),
```

Hence you don't pass it if you don't need it.

Reply:
Ah, yeah I always forget about Python idiosyncrasies for parameters.

Reply:
Decided to keep it as is for now. Let me know if you would rather have it any other way.

Reply:
Ok. Gathering quick feedback for deciding on one or the other. Will let the users decide. Don't mind either one.

Reply:
@Roland-djee Made the relevant changes. Changed to order of arguments and made an empty default for the FM.